torchbox/torchbox.com
# Tech debt - remove old settings

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1423005489)

### Description of Changes Made

- Remove an unused address setting (was the only option under 'Global settings', so that has gone)
- Remove an unused Tag class (was unused even in the old site)

### How to Test

- Check that the relevant settings are no longer visible in the wagtail admin
- Browse around the site and check nothing is broken

### Screenshots

<details>
  <summary>Expand to see more</summary>

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
